templates/swagger-ui/swagger.json

```diff
@@ -1086,6 +1071,186 @@
                   }
               ]
           }
-      }
+      },
+      "/capacity/s3/{resource}/{id}": {
+          "get": {
+            "tags": [
+                "Cortx Capacity"
+            ],
+            "summary": "Get user level Capacity usage statistics.",
```

Comment:
Summary is not looking good to me.  capacity info is based on the resource and id path params. lets say user provide resource=bucket, id=bucketid then this API will give bucket level stats.

Reply:
@rohitkolapkar 
I took the summary from REST API specification
Please suggest alternative

Reply:
updated